scripts/run_tests_unit.sh

```diff
@@ -19,5 +19,5 @@ function display_result {
   fi
 }
 
-py.test "$@"
+pipenv run py.test "$@"
```

Comment:
@warrenbailey we just use `pipenv run` in the terminal normally, but I suppose it makes no difference